go.sum

```diff
@@ -34,8 +34,8 @@ github.com/cyphar/filepath-securejoin v0.2.4/go.mod h1:aPGpWjXOXUn2NCNjFvBE6aRxG
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
-github.com/doublecloud/go-genproto v0.0.0-20240430040646-e4eb5018c1bf h1:bpAGwCaY9uRoLWFC1bC1WC79gIuHufdGoAPSBVqNKUQ=
-github.com/doublecloud/go-genproto v0.0.0-20240430040646-e4eb5018c1bf/go.mod h1:GaWzogQ0MCW4OjW16H1DsXiKOqHXqaYsMy0wKfXwoo4=
+github.com/doublecloud/go-genproto v0.0.0-20240521040605-29dded8070f8 h1:YMnvmk3LXmbBr7pE9MvbR9q89VxkktV+ee2qxTbnIks=
```

Comment:
i think you need rebase to latest master, i did merge my PR with federated auth already.

Reply:
sure